root/etc/cont-init.d/60-striptracks-init

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/with-contenv bash
+
+cat <<EOF
+----------------
+>>> Striptracks Mod by TheCaptain989 <<<
+Repo: https://github.com/linuxserver/docker-mods/tree/radarr-striptracks
+
+Version: $(cat /etc/version.tc989)
+----------------
+EOF
+
+# Determine if setup is needed
+if [ ! -f /usr/bin/mkvmerge ]; then
+  echo "Running first time setup."
+
+  if [ -f /usr/bin/apt ]; then
+    # Ubuntu
+    echo "Installing MKVToolNix using apt-get"
+    apt-get update && \
+    apt-get -y install mkvtoolnix && \
+    rm -rf /var/lib/apt/lists/*
+  elif [ -f /sbin/apk ]; then
+    # Alpine
+    echo "Installing MKVToolNix using apk"
+    apk add mkvtoolnix && \
```

Comment:
should be `apk add --no-cache mkvtoolnix`

Reply:
Fixed.